cedar-policy-core/src/ast/id.rs

```diff
@@ -176,7 +176,7 @@ impl<'de> Deserialize<'de> for Id {
     }
 }
 
-/// Deserialize a [`Name`] using `from_normalized_str`
+/// Deserialize a [`crate::ast::Name`] using `from_normalized_str`
```

Comment:
should this be, deserialize an `UnreservedId` using ...?

Reply:
make sense